pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1357,6 +1357,9 @@ type Worker struct {
 	// MachineControllerManagerSettings contains configurations for different worker-pools. Eg. MachineDrainTimeout, MachineHealthTimeout.
 	// +optional
 	MachineControllerManagerSettings *MachineControllerManagerSettings `json:"machineControllerManager,omitempty" protobuf:"bytes,19,opt,name=machineControllerManager"`
+	// Sysctl is a map of kernel settings to apply on all VMs in this worker pool.
```

Comment:
```suggestion
        // Sysctl is a map of kernel settings to apply on all VMs in this worker pool.
```

Reply:
Good catch. Thanks. I fixed the comments in the latest commit.